WalletWasabi.Tests/RegressionTests/CoinJoinTests.cs

```diff
@@ -261,8 +264,9 @@ public async Task CcjTestsAsync()
 			blindedOutputScriptsHash = new uint256(NBitcoin.Crypto.Hashes.SHA256(blindedData.BlindedOutput.ToBytes()));
 			proof = key.SignCompact(blindedOutputScriptsHash);
 			inputsRequest.Inputs.First().Proof = proof;
-			using (var aliceClient = await CreateNewAliceClientAsync(roundId, registeredAddresses, signerPubKeys, requesters, inputsRequest))
+
 			{
+				var aliceClient = await CreateNewAliceClientAsync(roundId, registeredAddresses, signerPubKeys, requesters, inputsRequest);
```

Comment:
Can you rename this to aliceClient1 and remove the brackets?

Reply:
It requires renaming more variables. That's something I'm not really eager to do as it makes the PR bigger and I don't understand the code there very well. We can file an issue for that.